### PR TITLE
Extract Hive to separate module - tempto-hive

### PR DESCRIPTION
**Pull request by maciejgrzybek**

### CODE REVIEW

File: tempto-hive/src/main/java/com/teradata/tempto/hive/fulfillment/table/HiveTableDefinition.java [hunk 1]
```diff
@@ -86,7 +82,7 @@ public String getCreateTableDDL(String name, Optional<String> location)
 
     public static HiveTableDefinition hiveTableDefinition(String name, String createTableDDLTemplate, HiveDataSource dataSource)
     {
-        return hiveTableDefinition(tableHandle(name), createTableDDLTemplate, dataSource);
+        return hiveTableDefinition(TableHandle.tableHandle(name), createTableDDLTemplate, dataSource);
```
Comment (cawallin) [inline tempto-hive/src/main/java/com/teradata/tempto/hive/fulfillment/table/HiveTableDefinition.java]: any reason to remove the static import?

File: tempto-hive/src/main/java/com/teradata/tempto/hive/fulfillment/table/HiveTableDefinition.java [hunk 2]
```diff
@@ -152,22 +148,22 @@ public HiveTableDefinitionBuilder setDataSource(HiveDataSource dataSource)
 
         public HiveTableDefinitionBuilder setNoData()
         {
-            return setDataSource(createStringDataSource(handle.getName(), NO_DATA_REVISION, ""));
+            return setDataSource(InlineDataSource.createStringDataSource(handle.getName(), NO_DATA_REVISION, ""));
```
Comment (cawallin) [inline tempto-hive/src/main/java/com/teradata/tempto/hive/fulfillment/table/HiveTableDefinition.java]: ditto on static imports

File: tempto-hive/src/main/java/com/teradata/tempto/hive/internal/fulfillment/table/HiveTableManager.java [hunk 3]
```diff
@@ -34,8 +34,6 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
-import static com.teradata.tempto.fulfillment.table.MutableTableRequirement.State.LOADED;
```
Comment (cawallin) [inline tempto-hive/src/main/java/com/teradata/tempto/hive/internal/fulfillment/table/HiveTableManager.java]: any reason to remove the static imports?

File: tempto-hive/src/main/resources/log4j.properties [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#
```
Comment (cawallin) [inline tempto-hive/src/main/resources/log4j.properties]: I didn't know you needed a separate log4j.properties file for each module, is this the case?

File: tempto-hive/src/main/java/com/teradata/tempto/hive/fulfillment/table/HiveTableDefinition.java (hunk 1 above)
Comment (maciejgrzybek) [inline tempto-hive/src/main/java/com/teradata/tempto/hive/fulfillment/table/HiveTableDefinition.java]: IntelliJ's bug. Will fix. Thx!

**Comment by maciejgrzybek** [comment]
Closing due to significant changes in https://github.com/prestodb/tempto/pull/176 (which this branch is based of). Will reopen once reimplemented.

**Pull request closed without merging**